assets/js/persistence.js

```diff
@@ -7,7 +7,7 @@ var URL_PARAM_Q_LIMIT = 1300,
     DEFAULT_EXPIRY_HOURS = 24;
 
 var store = new Store(config.HTML_URL);
-
+var hash, hashURLMap = {};
```

Comment:
Why is this in global scope if it's only being used inside the `cache` function?